tfutils/base.py

```diff
@@ -515,28 +657,33 @@ def run_base(saver_params,
             model_kwargs['cfg_initial'] = None
         if 'seed' not in model_kwargs:
             model_kwargs['seed'] = 0
-        train_outputs, cfg_final = model_func(inputs=train_inputs,
+            train_outputs, cfg_final = model_func(inputs=train_inputs,
```

Comment:
Something is going wrong with my editor causing TABs to be in places I did not intent them.  I'm going to have to check over the code line by line